cabin-sorting/hackerSortingAlgo.ts

```diff
@@ -9,8 +10,13 @@ let CABIN_SIZE: number
 let QUESTIONS_SIZE: number
 
 // Go to the given csv filepath and parse its contents into an array
-function loadCSV (filepath: string, headers: boolean): any[] {
```

Comment:
i'd rather we put I/O in a single file. let's move them out. 

additionally @mike10911 for any convenient time over the summer, i think to reduce some tech debt / make I/O even cleaner we can make abstract utils in a shared utils folder in `internal-tools`. could you please add this in to backlog?